magda-web-client/src/Components/Dataset/Add/DatasetLinkItem.tsx

```diff
@@ -255,16 +260,117 @@ const DatasetLinkItemEditing = (props: EditViewProps) => {
     );
 };
 
+async function getAllDataUrlProcessorsFromOpenfaasGateway() {
```

Comment:
Could use a JSDoc here - what's a data processor? How does one get it out of an open faas gateway?

Reply:
Will add comments --- probably includes a url to openfaas gateway API swagger file

Reply:
Added comments